src/interface_py/h2o4gpu/util/xgboost_migration.py

```diff
@@ -0,0 +1,34 @@
+def load_pkl(name):
+    """Load xgboost model from pickle and perform conversion from version 0.90 if require.
+
+    :return:
+        Total number of GPUs and total available memory
+    """
+    import pickle
+    import xgboost
+    with open(name, 'rb') as f:
+        try:
+            model = pickle.load(f)
+            return model
+        except xgboost.core.XGBoostError as e:
+            if "Check failed: header == serialisation_header_" in str(e):
+                import xgboost_prev
+                import tempfile
+
+                class Unpickler(pickle.Unpickler):
```

Comment:
I have a more thorough helper script written here: https://github.com/dmlc/xgboost/pull/5281/files  The PR is still WIP, but expect to be finished soon.  Sorry for the caused troubles.

Reply:
Thank you for your help.
Probably I just copy-paste sklearn's stuff.

The goal of the script is to allow smoothly make conversion without saving model on disk and spawning a new process.